namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/variables.tf

```diff
@@ -19,6 +19,34 @@ variable "team_name" {
   default     = "Digital-Prison-Services"
 }
 
+variable "additional_topic_clients" {
+  description = "Create a dedicated access key and store it in a secret named 'hmpps-domain-events-topic' in each of the below namespaces."
+  default = [
+    "activities-api-dev",
+    "calculate-release-dates-api-dev",
+    "court-probation-dev",
+    "hmpps-adjustments-dev",
+    "hmpps-assessments-dev",
+    "hmpps-community-accommodation-dev",
+    "hmpps-complexity-of-need-staging",
+    "hmpps-domain-event-logger-dev",
+    "hmpps-incentives-dev",
+    "hmpps-interventions-dev",
+    "hmpps-manage-offences-api-dev",
+    "hmpps-registers-dev",
+    "hmpps-restricted-patients-api-dev",
+    "hmpps-tier-dev",
+    "hmpps-workload-dev",
+    "make-recall-decision-dev",
+    "offender-case-notes-dev",
+    "offender-events-dev",
+    "offender-management-staging",
+    "offender-management-test",
+    "offender-management-test2",
+    "prisoner-offender-search-dev"
+  ]
```

Comment:
Anyone who needs access to the hmpps-domain-events topic just needs to add their namespace to this list, which will then create a dedicated access key and store it in a secret named "hmpps-domain-events-topic".

Reply:
There are a couple actually, e.g. https://github.com/ministryofjustice/cloud-platform-environments/blob/dedicated-access-keys-dev/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-activities-management.tf#L119-L130.

For those ones I've left the old secret in place, and created them a new one with the standard name.

Reply:
The ones with non-standard secret names are:
* activities-api-dev - "activities-domain-events-sqs-topic-instance-output"
* offender-case-notes-dev - "domain-events-topic"
* hmpps-workload-dev - "wmt-hmpps-domain-events-topic"

Reply:
I've created a ticket to rename the activities-api-dev secret to be consistent, and will remove the old one too. Thanks.